atmosphere/settings/__init__.py.j2

```diff
@@ -139,6 +142,10 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'atmosphere.slash_middleware.RemoveSlashMiddleware',
     'atmosphere.slash_middleware.RemoveCSRFMiddleware',
+    {% if ENABLE_DJANGO_PROMETHEUS %}
+    'django_prometheus.middleware.PrometheusAfterMiddleware',
+    {% endif %}
+
```

Comment:
Lets not do this here -- you can inject django_prometheus to the beginning/end of `MIDDLEWARE_CLASSES` in `atmosphere/settings/local.py.j2` [here is an example of adding to the end](https://github.com/cyverse/atmosphere/blob/master/atmosphere/settings/local.py.j2#L440-L443)